vaccine-standby/functions/setup.js

```diff
@@ -0,0 +1,507 @@
+            conditions: [
+              {
+                friendly_name: '1',
+                arguments: [
+                  '{{widgets.NotificationPreference.inbound.Body}}',
+                ],
+                type: 'equal_to',
+                value: '1',
+              },
+            ],
+          },
+          {
+            next: 'GetEmail',
+            event: 'match',
+            conditions: [
+              {
+                friendly_name: '2',
+                arguments: [
+                  '{{widgets.NotificationPreference.inbound.Body}}',
+                ],
+                type: 'equal_to',
+                value: '2',
+              },
+            ],
+          },
+        ],
+        properties: {
+          input: '{{widgets.NotificationPreference.inbound.Body}}',
+          offset: {
+            x: 0,
+            y: 2690,
+          },
+        },
+      },
+      {
+        name: 'GetEmail',
+        type: 'send-and-wait-for-reply',
+        transitions: [
+          {
+            next: 'LanguagePreference',
+            event: 'incomingMessage',
+          },
+          {
+            event: 'timeout',
+          },
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 300,
+            y: 2950,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your email address?',
+          timeout: '3600',
+        },
+      },
+      {
+        name: 'Goodbye',
+        type: 'send-message',
+        transitions: [
+          {
+            event: 'sent',
+          },
+          {
+            event: 'failed',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 60,
+            y: 3460,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          to: '{{contact.channel.address}}',
+          body: "Thank you, you are now now on the vaccine standby list.\n\nWe will notify you when you're eligible to receive a vaccine.",
+        },
+      },
+      {
+        name: 'LanguagePreference',
+        type: 'send-and-wait-for-reply',
+        transitions: [
+          {
+            next: 'Goodbye',
+            event: 'incomingMessage',
+          },
+          {
+            event: 'timeout',
+          },
+          {
+            event: 'deliveryFailure',
+          },
+        ],
+        properties: {
+          offset: {
+            x: 60,
+            y: 3210,
+          },
+          service: '{{trigger.message.InstanceSid}}',
+          channel: '{{trigger.message.ChannelSid}}',
+          from: '{{flow.channel.address}}',
+          body: 'What is your preferred language?\n1. English\n2. Español\n3. 繁體中文\n\nReply"1", "2", or "3"',
```

Comment:
What does this do? It doesn't seem like there's any other additional language support in this app